app/Http/Controllers/Apis/Protected/Summit/Strategies/events/RetrieveSummitEventsStrategy.php

```diff
@@ -248,6 +249,7 @@ protected function getFilterValidatorRules(): array
             'streaming_url' => 'sometimes|string',
             'etherpad_link' => 'sometimes|string',
             'streaming_type' => 'sometimes|string|in:VOD,LIVE',
+            'submission_status' => 'sometimes|string',
```

Comment:
we need to define possible values here
bc on db are NULL or Received
so i would said in:Received,NonReceived

Reply:
done